content/en/blog/_posts/2023-12-05-My-first-ever-KubeCon-KubeCon+CloudNativeCon-North-America-2023/index.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: blog
+title: "My first-ever KubeCon - KubeCon+CloudNativeCon North America 2023"
+date: 2023-12-05
+slug: My-first-ever-KubeCon-KubeCon+CloudNativeCon-North-America-2023
+---
+
+**Author -** Rajdeep Pal, Student, Techno India University
+
+_This post is a guide for beginners to look at huge and overwhelming events like KubeCon._
+
+# Introduction
+
+![KubeCon](https://raw.githubusercontent.com/kubernetes/website/main/content/en/blog/_posts/2023-12-05-My-first-ever-KubeCon-KubeCon+CloudNativeCon-North-America-2023/pic1.jpg)
+
+This blog is kind of a personal experience cum guide for beginners when they first encounter such a huge and important conference. Although KubeCon is the first conference I attended, it is certainly the largest. Earlier I have been to community-driven conferences such as Google Cloud Community Days Kolkata 2023, Kubernetes Community Days Mumbai 2023, and KubeDay India 2023, they were quite big too but they lasted for a day or two. KubeCon was much larger and more overwhelming for new folks in the field. I attended my first KubeCon virtually as the Dan Kohn Scholarship approved me for virtual attendance. Before deep-diving into the event and my experiences, let's start with the basics of the event or any event hosted by the Cloud Native Computing Foundation.
+
+Like any other event, you need to register at their official site. You can learn about upcoming events on their [official page](https://www.cncf.io/events/). Sometimes it might be not easy to afford the cost of attending these events, but CNCF also offers scholarships to individuals. To provide an overview, CNCF mainly offers scholarships based on 3 categories namely - Diversity, Need-Based, and Maintainer.
+
+![KubeCon](https://raw.githubusercontent.com/kubernetes/website/main/content/en/blog/_posts/2023-12-05-My-first-ever-KubeCon-KubeCon+CloudNativeCon-North-America-2023/pic2.png)
+
+KubeCon serves as a dynamic platform for attendees to actively engage with the expansive Kubernetes community. Through a diverse array of networking avenues, including Birds of a Feather session, social events, and dedicated breaks, participants have the opportunity to connect with fellow developers, contributors, and users of Kubernetes. These interactions not only foster collaboration but also facilitate the exchange of knowledge and experiences. The conference's emphasis on networking creates an environment where like-minded professionals and industry experts can forge connections, share insights, and collectively contribute to the ever-evolving landscape of container orchestration and cloud-native technologies.
+
+# General Information about KubeCon
+
+This year's KubeCon North America 2023 was held from November 6th to 9th of 2023 in Chicago, Illinois.
+
+![KubeCon](https://raw.githubusercontent.com/kubernetes/website/main/content/en/blog/_posts/2023-12-05-My-first-ever-KubeCon-KubeCon+CloudNativeCon-North-America-2023/pic3.jpg)
```

Comment:
{{< figure >}} shortcode would be nice

Reply:
I guess something is not going right. The pics are not loading

Reply:
{{< figure src="https://github.com/Rajdeep1311/website/blob/main/content/en/blog/_posts/2023-12-05-My-first-ever-KubeCon-KubeCon+CloudNativeCon-North-America-2023/pic1.jpg" alt="KubeCon NA 2023 Poster" >}}

Is the syntax right?

{{< figure src="pic1.jpg" alt="KubeCon NA 2023 Poster" >}}
Is too does not work

Reply:
> {{< figure src="pic1.jpg" alt="KubeCon NA 2023 Poster" >}}

This is correct, I suggest checking the local preview again; (I've checked a preview from your fork and it works fine)